src/test/java/org/elasticsearch/cluster/BlockClusterStatsTests.java

```diff
@@ -41,7 +44,7 @@
 
```

Comment:
Yes, everything is covered in other test classes. I also think we can removed it now.